targets/TARGET_NXP/TARGET_LPC176X/device/flash_api.c

```diff
@@ -16,7 +16,6 @@
 
 #include "mbed_critical.h"
 
-#if DEVICE_FLASH
```

Comment:
if this was removed, then one more #endif . However, should this be left in there?